### PR TITLE
More doc updates for dedicated namespace

### PR DESCRIPTION
**Pull request by doxiao**
Add a little more doc content, and fix minor doc issues.

### CODE REVIEW

File: docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md [hunk 1]
```diff
@@ -181,6 +181,10 @@ These examples show two valid YAML syntax options for arrays.
 You must include the `default` namespace in the list if you want the operator to monitor both the `default` namespace and some other namespaces.
 {{% /notice %}}
 
+{{% notice note %}}
+This value is ignored if `dedicated` is set to `true`. The operator will then only manage domains in its own namespace.
```
Comment (rosemarymarano) [inline docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md]: The operator will then only manage domains -> Then, the operator will manage only domains

Comment (doxiao) [reply docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md]: done. thanks.

Comment (doxiao) [reply docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md]: > Slight re-wording of one sentence.

Updated.

Please see a newly added section in docs-source/content/security/service-accounts.md .  Thanks.

**Comment by bsmoyers** [comment]
Can one of the admins verify this patch?

**Pull request merged**